packages/clerk-js/src/utils/__tests__/devbrowser.test.ts

```diff
@@ -48,8 +53,13 @@ describe('getDevBrowserJWTFromURL(url,)', () => {
     ['#foo__clerk_db_jwt[deadbeef]', 'deadbeef', '#foo'],
     ['/foo?bar=42#qux__clerk_db_jwt[deadbeef]', 'deadbeef', '/foo?bar=42#qux'],
   ];
+
   test.each(testCases)('returns the dev browser JWT from a url. Params: url=(%s), jwt=(%s)', (url, jwt, calledWith) => {
     expect(getDevBrowserJWTFromURL(url)).toEqual(jwt);
-    calledWith && expect(replaceStateMock).toHaveBeenCalledWith(null, '', calledWith);
+    if (calledWith === null) {
+      expect(replaceStateMock).not.toHaveBeenCalled();
+    } else {
+      expect(replaceStateMock).toHaveBeenCalledWith(null, '', calledWith);
+    }
```

Comment:
🙃 not a fun of conditions in the tests. I would expect to have a separate test case with a distinct name instead of `if / else` .

Reply:
That makes sense!
I'll update the test accordingly 🙂